source/triggers/database-triggers.txt

```diff
@@ -234,6 +234,20 @@ options are available:
 
      - .. include:: /includes/trigger-auto-resume.rst
 
+   * - :guilabel:`Maximum Throughput Triggers`
+
+     - .. _triggers-maximum-throughput:
+      
+       If your App Services app is running on a dedicated server (M10+ Tier), 
+       you can increase the maximum throughput from the default 10K to 50K. 
```

Comment:
Can we link to the [Event Processing Throughput](https://www.mongodb.com/docs/atlas/app-services/triggers/#event-processing-throughput) limitation?

Also we don't define throughput on this page - can we clarify that 10K and 50K refer to the maximum number of events that can be processed simultaneously